### DISCUSSION TITLE
Sort by Title?

### DISCUSSION THREAD

**Discussion by dcj72** [omnivore-app/omnivore#3795] [Q&A]
Hi!

New to omnivore -- is there a way to sort by title?  Am I being dopey and just missing something?

Thanks!

**Comment by IamSmeagol** [comment]
I don't think it's possible to sort the article list by title on Android or web. 

Could you let us know what platform you are using?

**Reply by dcj72** [reply]
I'm using the web.  Would be nice to have a few extra sort options, though I'm sure the developers are inundated with feature requests. :)
